appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -372,6 +378,7 @@ public UserResource toResource(UserDTO user)
 					addWeekActivityOverviewsLink(userResource);
 					addNewDeviceRequestLink(userResource);
 					addAppActivityLink(userResource);
+					pinResetRequestController.addLinks(userResource);
```

Comment:
The reason for doing it this way is that otherwise the knowledge about which link type are appropriate in what situation would be spread across the user controller and the pin reset request controller. For the other links, the condition (mobile number confirmed) is know to the user controller.